packages/governance/src/committeeRegistrar.js

```diff
@@ -1,49 +1,71 @@
 // @ts-check
 
-import { Far } from '@agoric/marshal';
-import { makeNotifierKit } from '@agoric/notifier';
 import { E } from '@agoric/eventual-send';
-import { makeStore } from '@agoric/store';
+import { Far } from '@agoric/marshal';
+import { makeSubscriptionKit } from '@agoric/notifier';
 import { allComparable } from '@agoric/same-structure';
+import { makeStore } from '@agoric/store';
+import { natSafeMath } from '@agoric/zoe/src/contractSupport/index.js';
+
+import { makeHandle } from '@agoric/zoe/src/makeHandle';
+import { QuorumRule } from './question.js';
+
+const { ceilDivide } = natSafeMath;
 
 // Each CommitteeRegistrar represents a particular set of voters. The number of
 // voters is visible in the terms.
+//
+// This contract creates an electorate that is not visible to observers. There
+// may be uses for such a structure, but it is not appropriate for elections
+// where the set of voters needs to be known, unless the contract is used in a
+// way that makes the distribution of voter facets visible.
+
+/** @type {ContractStartFn} */
```

Comment:
With a small tweak, the tools will understand that this comment is connected to `start`:

```suggestion
/**
 * Each CommitteeRegistrar represents a particular set of voters. The number of
 * voters is visible in the terms.
 *
 * This contract creates an electorate that is not visible to observers. There
 * may be uses for such a structure, but it is not appropriate for elections
 * where the set of voters needs to be known, unless the contract is used in a
 * way that makes the distribution of voter facets visible.
 *
 * @type {ContractStartFn}
 */
```

Reply:
I'll put it there, but I'm not sure that matches my intentions. I thought of it as file-level comment. In JavaDoc, it would have been a class comment.  If there were additional things defined in this file, this comment wouldn't apply to `start` in particular.